Makefile

```diff
@@ -81,12 +81,24 @@ simpy_cpython:
 		--exclude "*/bad*" \
 		--exclude "*/lib2to3/tests/data/*"
 
+cpython:
```

Comment:
Nit (feel free to ignore): Maybe `unpack-cpython`?

Reply:
No, I want this to be the name of the cpython directory so make will extract the tarball only if needed. (But I can't make it depend on the tarball, else make will always extract it because tar sets the mtime of the extracted directory to whatever's in the tarball.)

FWIW our tarball contains a lot of extras.